Plugins/DeleteUnusedBranches/DeleteUnusedBranchesForm.cs

```diff
@@ -212,15 +215,29 @@ private void Refresh_Click(object sender, EventArgs e)
             ThreadHelper.JoinableTaskFactory.RunAsync(() => RefreshObsoleteBranchesAsync());
         }
 
+        private void CheckBoxHeader_OnCheckBoxClicked(bool state)
+        {
+            for (int i = 0; i < BranchesGrid.Rows.Count; i++)
+            {
+                DataGridViewRow row = BranchesGrid.Rows[i];
+                DataGridViewCheckBoxCell cell =
+                    (DataGridViewCheckBoxCell)row.Cells[nameof(deleteDataGridViewCheckBoxColumn)];
+                cell.Value = state;
+            }
+
+            BranchesGrid.EndEdit();
```

Comment:
why is this needed and where is `BeginInit` call?

Reply:
It won't work otherwise. Added `CommitEdit(...)` call before.